src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const money = days * 40;
+
+  if (days < 3) {
+    return money;
+  }
+
+  if (days >= 3 && days < 7) {
+    return money - 20;
```

Comment:
move this numbers to variables, to everyone be able to understand where they came from